src/components/pages/InstructorHome/index.js

```diff
@@ -16,7 +15,7 @@ const InstructorHome = props => {
   const { courses } = props;
   const history = useHistory();
   const dispatch = useDispatch();
-  const idToken = useOktaAuth().oktaAuth.getIdToken();
+  const idToken = "stub";
```

Comment:
Instead of "stub", another TO-DO comment and maybe "Auth0" might be better to find later when replacing.

Reply:
That's a great idea, thank you!